packages/application-shell/src/components/navbar/navbar.tsx

```diff
@@ -79,6 +88,15 @@ const getMenuVisibilityOfMainmenu = (menu: TNavbarMenu) =>
   menu.menuVisibility ? [menu.menuVisibility] : [];
 
 const ApplicationMenu = (props: ApplicationMenuProps) => {
+  const [topPosition, setTopPosition] = useState(0);
+  const elementRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (elementRef.current != null) {
+      setTopPosition(elementRef.current.offsetTop);
```

Comment:
I guess it's ok for the first iteration, I've seen some corner cases
<img width="355" alt="Screenshot 2023-08-23 at 12 12 22" src="https://github.com/commercetools/merchant-center-application-kit/assets/49066275/e925787e-41b7-42f0-b1a4-de9cefe68008">

Reply:
Sure,
![2023-08-24 09 29 02](https://github.com/commercetools/merchant-center-application-kit/assets/49066275/572ba2c8-c840-4bd6-9031-63db8eef2766)
It's when the navbar is being scrolled and menu item is half-visible.
However, I believe this isn't a matter of concern for this PR.